### ISSUE TITLE
New site address

### ISSUE THREAD

**Issue by SoxFace** [JakeStoeffler/code_embed_lti#6]
Hi Jake, 

Thanks for all your hard work on this LTI app. It's a great way to help us newbies, to get started in LTI. The README links to the app are dead, but I've found a live version here:

https://code-embed-lti.herokuapp.com/

I might do a pull request to update your README, keep an eye out.

Cheers
Sonya

**Comment by JakeStoeffler** [comment]
@SoxFace Yeah, my domain expired but the app is still live on Heroku so that URL should continue to work unless they shut it down. I'd be happy to make you an admin on that if you want to help manage it. Let me know.

Feel free to make a PR and I will try to get it merged ASAP.

Thanks!
